app/main.ipynb

```diff
@@ -1,312 +1,469 @@
-   "source": [
-    "# write your code for task 6 here"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "# Task 7\n",
-    "Compare the married and non-married men salaries. Who earns more? (>50K or <=50K)\n",
-    "Married men are those, whom `marital-status` starts with \"Married\". Others are not."
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   }
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
-   "source": [
-    "# write your code for task 7 here"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "# Task 8\n",
-    "Get the max hours per week some person works. How many people works the same amount of hours per week?"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   }
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
-   "source": [
-    "# write your code for task 8 here"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "# Task 9\n",
-    "Analyze the correlation between data in dataset. Understand connected fields in it and print highlight thier connection."
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   }
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 7,
-   "outputs": [],
-   "source": [
-    "# write your code for task 9 here"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  }
- ],
- "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 2
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython2",
-   "version": "2.7.6"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 0
-}
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Task 0\n",
+    "Read the dataset from csv file & perform data cleaning - remove all rows, which contains `?` in some columns.\n",
+    "Also check for data correctness (salary & salary $K)."

```

Comment:
Maybe also need to check if all is right?

Reply:
> Maybe also need to check if all is right?

I don't understand how can I do this, like df.info()?

Reply:
Check salary and salary $K colums. If value of salary column is "<=50K", then salary $K value should be <= 50.
The same for >50K

Reply:
> Check salary and salary $K colums. If value of salary column is "<=50K", then salary $K value should be <= 50. The same for >50K

Thanks for the explanation, I finally understood what was required)